apps/frontend/src/components/cards/controltable/ControlRowDetails.vue

```diff
@@ -189,7 +189,7 @@ export default class ControlRowDetails extends mixins(HtmlSanitizeMixin) {
   }
 
   get errorMessage(): string {
-    return this.control.hdf.segments?.length == 0
+    return this.control.root.hdf.segments?.length == 0
```

Comment:
The old guy still has good instincts :)

Reply:
Just curious: how does it determine what is a "root" profile object in the results json? Is it based on any that do not contain a dependency?